tests/common/fixtures/consistency_checker/consistency_checker.py

```diff
@@ -0,0 +1,335 @@
+import pytest
+import logging
+import json
+import os
+import datetime
+from collections import defaultdict
+
+logger = logging.getLogger(__name__)
+
+SYNCD_CONTAINER = "syncd"
+QUERY_ASIC_SCRIPT = "query-asic.py"
+LIBSAIREDIS_DEB = "libsairedis.deb"
+PYTHON3_PYSAIREDIS_DEB = "python3-pysairedis.deb"
+DUT_DST_PATH_HOST = "/tmp/consistency-checker"
+DUT_DST_PATH_CONTAINER = "/consistency-checker"
+
+DUT_SCRIPT_PATH_SRC = os.path.dirname(__file__) + "/" + QUERY_ASIC_SCRIPT
```

Comment:
`DUT_SCRIPT` is too generic a name. Can it be renamed to something like `QUERY_ASIC_SCRIPT_PATH_DST_CONTAINER `